### DISCUSSION TITLE
Problem with accounts

### DISCUSSION THREAD

**Discussion by GPAS2022** [community/community#69339] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

So I'm triyng to log in with an account using the username, but the after i insert the unsername and password it asked me to insert the code verification that send me but first of all the email  is different from one letter and I don't know how to resolve this problem, anyone in my situation?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GPAS2022 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
